sdks/java/extensions/google-cloud-platform-core/src/test/resources/test-hadoop-conf.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
```

Comment:
Could you add license information here? Example: https://github.com/apache/beam/blob/e1e093455acaa9a6d5b131cf5103165a661c7615/sdks/java/build-tools/src/main/resources/beam/checkstyle/checkstyle.xml#L2

Reply:
Ended up removing the file